src/wallet/wallet_rpc_server.cpp

```diff
@@ -34,6 +34,7 @@
 #include <boost/preprocessor/stringize.hpp>
 #include <cstdint>
 #include "include_base_utils.h"
+#include "wallet_rpc_server_error_codes.h"
```

Comment:
Why this change?

Reply:
first commit i added an error message and the WRONG_ADDRESS core rpc error. Clangd automatically included its header. forgot to remove it